tests/unit/CookieJar.spec.ts

```diff
@@ -41,6 +41,7 @@ const DUMMY_WALLET_OBJECTS: WalletInfoEntry[] = [
         ]),
         contracts: [],
         type: WalletType.BIP39,
+        deleted: false,
```

Comment:
I think we should add encoding this flag to the CookieJar and test it here, but that can be a separate issue.

Reply:
I put the first two `WalletInfoEntry` to `keyMissing=false,` and the last one to `keyMissing=true,` and Updated `BYTES` and `OUT_DUMMY_WALLET_OBJECTS` accordingly.